src/components/LinkListItem.js

```diff
@@ -51,31 +50,59 @@ export default class LinkListItem extends React.Component {
             classes.push("has-right-icon");
         }
 
+        if (this.props.to.startsWith('mailto')) {
```

Comment:
Propose instead we create a second property called href. Which I think `<Link>` already supports?